src/controllers/WhoWasIllController.ts

```diff
@@ -0,0 +1,18 @@
+import { SessionMiddleware } from 'ch-node-session-handler';
+import { controller, httpGet } from 'inversify-express-utils';
+
+import { BaseAsyncHttpController } from 'app/controllers/BaseAsyncHttpController';
+import { FeatureToggleMiddleware } from 'app/middleware/FeatureToggleMiddleware';
+import { Feature } from 'app/utils/Feature';
+import { WHO_WAS_ILL_PAGE_URI } from 'app/utils/Paths';
+
+const template = 'illness/who-was-ill';
+
+@controller(WHO_WAS_ILL_PAGE_URI, FeatureToggleMiddleware(Feature.ILLNESS_REASON), SessionMiddleware)
```

Comment:
Why is there session middleware applied to it?

Reply:
Good question, I'll remove.